pom.xml

```diff
@@ -708,6 +711,31 @@
         <type>tar.gz</type>
       </dependency>
 
+      <dependency>
+        <groupId>org.commonjava.indy</groupId>
+        <artifactId>indy-subsys-metrics</artifactId>
+        <version>1.2.0-SNAPSHOT</version>
+      </dependency>
+      <dependency>
```

Comment:
Let's just combine all of these submodules into indy-subsys-metrics. We're doing similar things for indy-subsys-groovy, where one subsystem contains a few functions that are used throughout Indy.